### PR TITLE
nsxv: stylefixes for bash

### PR DESCRIPTION
**Pull request by rkhozinov**

### CODE REVIEW

**Comment by rkhozinov** [comment]
@naumvd95 please check that

File: servers/partner-ci/builders/nsxv.test.sh [hunk 1]
```diff
@@ -58,28 +58,26 @@ fi
...
-echo "plugin-checksum: $(md5sum -b ${NSXV_PLUGIN_PATH})"
+echo -e "test-group: ${TEST_GROUP}\n \
+        env-name: ${ENV_NAME}\n \
+        use-snapshots: ${USE_SNAPSHOTS}\n \
+        fuel-release: ${FUEL_RELEASE}\n \
+        venv-path: ${VENV_PATH}\n \
+        env-name: ${ENV_NAME}\n \
+        iso-path: ${ISO_PATH}\n \
+        plugin-path: ${NSXV_PLUGIN_PATH}\n \
+        plugin-checksum: $(md5sum -b ${NSXV_PLUGIN_PATH})\n"
```
Comment (naumvd95) [inline servers/partner-ci/builders/nsxv.test.sh:77]: seems unnecessary space?

File: servers/partner-ci/builders/nsxv.test.sh [hunk 2]
```diff
@@ -108,5 +106,5 @@ if [[ "${DEBUG}" == "true" ]]; then
   sudo cp /var/log/libvirt/libvirtd.log ${WORKSPACE}/libvirtd_after_test.log
   sudo chown jenkins:jenkins ${WORKSPACE}/libvirtd_after_test.log
 fi
-echo "ENVIRONMENT NAME is $ENV_NAME"
+
```
Comment (naumvd95) [inline servers/partner-ci/builders/nsxv.test.sh:105]: why do u remove it?

**Review by naumvd95 — CHANGES_REQUESTED** [review]

File: servers/partner-ci/builders/nsxv.test.sh (hunk 2 above)
Comment (rkhozinov) [reply servers/partner-ci/builders/nsxv.test.sh:105]: it duplicates action above ( see `echo -n "..."`)

Comment (rkhozinov) [reply servers/partner-ci/builders/nsxv.test.sh:105]: please fetch the branch and run shellcheck

Comment (naumvd95) [reply servers/partner-ci/builders/nsxv.test.sh:105]: done

**Pull request merged**